app/workers/delete_work_history_audits_worker.rb

```diff
@@ -0,0 +1,16 @@
+class DeleteWorkHistoryAuditsWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :low_priority
+
+  AUDITABLE_TYPES = %w[ApplicationExperience ApplicationWorkHistoryBreak].freeze
+
+  def perform(audit_ids)
+    Audited::Audit.where(id: audit_ids).find_each(batch_size: 100) do |audit|
+      audit.delete if AUDITABLE_TYPES.include?(audit.auditable_type) &&
+                      audit.associated_type == 'ApplicationChoice' &&
+                      audit.action == 'create' &&
+                      audit.username == '(Automated process)'
```

Comment:
It looks like workers add 'Automated process' as username username https://github.com/DFE-Digital/apply-for-teacher-training/blob/0013b1b33d9ca9fdaec6b13c07822488f9de882c/lib/workers/audit_trail_attribution_middleware.rb#L6